tests/rt.conf

```diff
@@ -128,8 +128,9 @@ RUN     | fv3_ccpp_esg_HAFS_v0_hwrf_thompson_debug
 # CPLD tests                                                                                                                                                                      #
 ###################################################################################################################################################################################
 
-COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16beta_coupled S2S=Y           | - wcoss_cray gaea.intel jet.intel       | fv3 |
+COMPILE | SUITES=FV3_GFS_2017_coupled,FV3_GFS_2017_satmedmf_coupled,FV3_GFS_v15p2_coupled,FV3_GFS_v16beta_coupled S2S=Y WW3=Y     | - wcoss_cray gaea.intel jet.intel       | fv3 |
```

Comment:
This makes the compile the same as line 162, which makes me wonder if we should either move the new test w/waves under that compile or just remove the second compile?

Reply:
@JessicaMeixner-NOAA I fixed them. Thanks.